docs/Encapsulation.md

```diff
@@ -51,9 +51,7 @@ fastify.register(async function authenticatedContext (childServer) {
     method: 'GET',
     handler (request, response) {
       response.send({
-        answer: request.answer,
-        foo: request.foo,
-        bar: request.bar
+        answer: request.answer
```

Comment:
The point of including the decorators here is to show that they are not available and will return as `undefined`.